scripts/utils/options.ts

```diff
@@ -29,7 +29,8 @@ export type BaseOption = {
 export type BooleanOption = BaseOption & {
   type: 'boolean';
   /**
-   * Does this option default true?
+   * If this option is set to true and the option value is false or undefined, the flag `--no-option` will be set.
```

Comment:
Should this say "if `inverse` is set to true and the option value is ..." -- I think there are two "option"s in this sentence and that's pretty confusing ;)